diff --git a/node_modules/rn-range-slider/index.tsx b/node_modules/rn-range-slider/index.tsx
index 37163ae..b9b21bd 100644
--- a/node_modules/rn-range-slider/index.tsx
+++ b/node_modules/rn-range-slider/index.tsx
@@ -188,12 +188,11 @@ const Slider: React.FC<SliderProps> = ({
   const {panHandlers} = useMemo(
     () =>
       PanResponder.create({
-        onStartShouldSetPanResponderCapture: falseFunc,
+        onStartShouldSetPanResponderCapture: trueFunc,
         onMoveShouldSetPanResponderCapture: falseFunc,
         onPanResponderTerminationRequest: falseFunc,
         onPanResponderTerminate: trueFunc,
         onShouldBlockNativeResponder: trueFunc,
-
         onMoveShouldSetPanResponder: (
           evt: GestureResponderEvent,
           gestureState: PanResponderGestureState,
